lms/djangoapps/certificates/views/webview.py

```diff
@@ -20,6 +20,7 @@
 from edxmako.shortcuts import render_to_response
 from edxmako.template import Template
 from eventtracking import tracker
+from national_id.models import ExtraInfo
```

Comment:
@pepeportela Last version of `feature-14162-ask-for-DNI` has class NationalId instead of ExtraInfo. Fix this.